charts/gateway-helm/values.tmpl.yaml

```diff
@@ -26,6 +26,10 @@ deployment:
     labels: {}
 
 config:
+  # -- Switch to false to prevent creating the configmap
+  enabled: true
```

Comment:
This looks weird to me, what problems are we trying to solve?

Reply:
some users may not want the EG Helm Chart to create the configmap, but would like to hand write it themselves, but would still want the EG Deployment to mount that user defined configmap